src/main/java/randoop/test/RegressionCaptureGenerator.java

```diff
@@ -253,4 +277,44 @@ public RegressionChecks generateTestChecks(ExecutableSequence sequence) {
     }
     return checks;
   }
+
+  /**
+   * Returns true if the method:
+   *
+   * <ul>
+   *   <li>has one formal parameter
+   *   <li>has a return type of primitive, boxed primitive, String, or enum
+   *   <li>is side-effect-free
+   * </ul>
+   *
+   * @param tco side-effect-free method as a TypedClassOperation
+   * @return whether we can use this method as a side-effect-free assertion
```

Comment:
This is better than the main method comment above.  Make the two consistent.
In several places the code uses "as an assertion", but you really mean "in an assertion".  For a method (call) to be as an assertion, its return type would have to be boolean.